_data/tools.yml

```diff
@@ -256,6 +256,18 @@
   v3: true
   v3_1: true
 
+- name: OpenAPI-UI
+  category: documentation
+  description:
+    a simpler and more beautiful openapi UI document than swagger UI, 
+    quickly generate mock params and call api request.
+  link: https://docs.openapi-ui.com
+  github: https://github.com/rookie-luochao/openapi-ui
+  language: React.js
+  v2: true
+  v3: true
+  v3_1: true
```

Comment:
Can you confirm it 100% works with the `3.1` standard?

Reply:
support for 3.1 is not comprehensive and needs to be improved while using it

Reply:
Ok I advise setting this to be false for now, can always submit a new PR to update this later.

Reply:
This is a good suggestion, Could you please provide a Commit suggestion? Thanks

Reply:
```suggestion
  v3_1: false
```